### DISCUSSION TITLE
2023-05-29 질문입니다.

### DISCUSSION THREAD

**Discussion by park7628** [park7628/ARUnityProject#32] [Q&A]
### 핵심 : 다른 씬에 있는 데이터를 활용하고 싶습니다.

내용 : 전체적인 UI를 구성한 씬에서 AR실험으로 넘어갔다가 다시 UI씬으로 넘어가는 씬전환을 했는데 가장 첫 번째 로그인 페이지로 돌아가는 상황이 생겼습니다. 스크립트에 저장되어있는 데이터들을 가져와서 넘어 가야되는 화면으로 설정하고 싶은데 어떻게 하면 좋을까요?

@jongfeel

**Comment by jongfeel** [comment]
우선
빠르게 해결해 볼 수 있는 방법으로 여러가지가 있는데 DontDestroyOnLoad 함수를 쓰는 게 있습니다.
https://docs.unity3d.com/ScriptReference/Object.DontDestroyOnLoad.html

그런데 추천하는 방법은 아니라서
Firebase를 써서 데이터를 가져올 수 있다는 가정하에 해보시는 걸 추천드립니다.

하지만 아직 서버가 안된다는 가정을 해보면 필요한 데이터를 파일에 저장해 두고 scene 전환 후에 읽어오는 방법도 있습니다.
그 파일을 저장해 두는 경로는 persistantDataPath라고 있는데 아래 링크를 참조해서 쓰면 됩니다.
https://docs.unity3d.com/ScriptReference/Application-persistentDataPath.html
